### PR TITLE
Add Segment.io

### PR DESCRIPTION
**Pull request by eordano**

### CODE REVIEW

File: src/utils/analytics.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+/// List of tracked events
+///
+
+export const sceneCreated = track('Scene created');
+export const preview = track('Preview started');
+export const sceneUploadRequested = track('Scene upload started');
+export const sceneUploaded = track('Scene uploaded');
+export const sceneLink = track('Scene ethereum link started');
+export const sceneLinked = track('Scene ethereum linked');
+export const deploy = track('Scene deploy requested');
```
Comment (cazala) [inline src/utils/analytics.ts]: what's the difference between the `Scene deploy requested` and the `Scene upload started` events?

File: src/utils/uploader.ts [hunk 2]
```diff
@@ -105,6 +109,9 @@ export async function uploader(vorpal: any, args: any, callback: () => void) {
     vorpal.log('Updating IPNS reference to folder hash... (this might take a while)');
     const { name } = await ipfsApi.name.publish(ipfsHash, { key: project.id });
 
+    // Upload successful
+    sceneUploaded();
```
Comment (matiasbargas) [inline src/utils/uploader.ts]: have sense to send the ipns name to the track properties?

Comment (matiasbargas) [reply src/utils/uploader.ts]: same with scenes that are being linked

File: src/utils/analytics.ts (hunk 1 above)
Comment (eordano) [reply src/utils/analytics.ts]: `push` vs `upload` -- hmm maybe not necessary I'll work on simplifying this

File: src/utils/uploader.ts (hunk 2 above)
Comment (eordano) [reply src/utils/uploader.ts]: Roger

**Pull request closed without merging**